deepliif/models/networks.py

```diff
@@ -340,7 +347,7 @@ def __init__(self, input_nc, output_nc, ngf=64, norm_layer=nn.BatchNorm2d, use_d
         else:
             use_bias = norm_layer == nn.InstanceNorm2d
 
-        model = [nn.ReflectionPad2d(3),
+        model = [nn.ZeroPad2d(3) if os.environ['DEEPLIIF_SEED'] != 'None' else nn.ReflectionPad2d(3),
```

Comment:
Please use the new version of the networks.py. I just updated the file and added padding_type to define_G and Resnet_Generator. If the padding_type is 'reflect' we use the ReflectionPad2d otherwise, we use ZeroPad2d. Instead of checking the DEEPLIIF_SEED, we can check the padding_type parameter. I believe it would make more sense and give more flexibility to the user, meaning that even without using the deterministic version, they can use the ZeroPad2d by only setting the padding_type to 'zero'. We can change the padding_type to 'zero' in the set_seed() function to make sure that the user will not receive any error by forgetting to set the padding_type to 'zero'.